src/open_clip/factory.py

```diff
@@ -141,9 +142,11 @@ def create_model(
                 assert False, 'pretrained image towers currently only supported for timm models'
 
         cast_dtype = get_cast_dtype(precision)
-        custom_text = model_cfg.pop('custom_text', False) or force_custom_text or ('hf_model_name' in model_cfg['text_cfg'])
+        custom_text = model_cfg.pop('custom_text', False) or force_custom_text or ('hf_model_name' in model_cfg.get('text_cfg', {}))
 
         if custom_text:
+            if 'hf_model_name' in model_cfg.get('text_cfg', {}):
```

Comment:
why this change

Reply:
To be sure its an huggingface model. I don't want to set unrelated parameters for model configs that aren't using HF.
Otherwise you could land in this branch if
`model_cfg['custom_text']` or `force_custom_text` is True.